src/translations/en.json

```diff
@@ -30,5 +30,20 @@
     "unready": "UNREADY",
     "exit": "EXIT",
     "invite": "Invite"
+  },
+  "ruleText":{
+    "1": "Scrap all points",
+    "2": "Scrap target Royal or Glasses eight",
+    "3": "Choose 1 card in the Scrap and put it to your hand",
+    "4": "Your opponent discards two cards of their choice from their hand",
+    "5": "Draw two cards from the deck",
+    "6": "Scrap all Royals and Glasses eights",
+    "7": "Play one of the top two cards of the deck and put the other back (both are revealed)",
+    "8": "Your opponent plays with an open hand (their cards are revealed to you)",
+    "9": "Return target card to its controller's hand. They can't play it next turn",
+    "10": "No effect",
+    "11": "Play on top of target point card to steal it",
+    "12": "Your other cards may only be targeted by scuttles",
+    "13": "Reduces the points you need to win. (1K: 14pts, 2K: 10pts, 3K: 5pts, 4K: 0pts)"
```

Comment:
I thought it better to put it inside the `translations` files, this way both the back and front end can both read from the same spot. By doing this I set up the initial localization of the dialog components that use the rule text.

Reply:
I think that makes sense. I think this placement at the top level is okay as well, though we could consider putting them inside "global". It's got enough stuff that it seems reasonable to keep it at the top level, idk